src/libs/ReportUtils.js

```diff
@@ -772,6 +772,9 @@ function getDisplayNameForParticipant(login, shouldUseShortForm = false) {
         return '';
     }
     const personalDetails = getPersonalDetailsForLogin(login);
+    if (!personalDetails) {
+        return '';
```

Comment:
We should probably return `login` instead of an empty string. That should be defined in all the places that we pass it in as a function param

Reply:
That's right, at least it won't be empty. I actually think we should even run `Str.removeSMSDomain(personalDetails.login);` with the login instead and then do the other stuff if the details exist